monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ComponentControllerV1.java

```diff
@@ -210,18 +213,41 @@ public OutputColumns getOutputColumns(@PathVariable(PARAMETER_NAME_TENANT) final
     @RequestMapping(value = "/{name}", method = RequestMethod.PUT, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
     public ProcessStatelessOutput processStateless(@PathVariable(PARAMETER_NAME_TENANT) final String tenant,
             @PathVariable(PARAMETER_NAME_NAME) final String name,
+            @RequestParam(value = PARAMETER_NAME_OUTPUT_STYLE, required = false, defaultValue = "tabular") OutputStyle outputStyle,
```

Comment:
Guys, is there a way how to use a enum constant here for the annotation attribute "defaultValue"? I tried also creating a static final String "constant" initialized with the enum.name(), but then compiler still complaints that "Attribute value must be constant".

I guess that compiler is clever enough to recignize that even static final field is not a real constant if it is initialized with a method call...